app/models/miq_server/at_startup.rb

```diff
@@ -3,13 +3,31 @@ module MiqServer::AtStartup
   include Vmdb::Logging
 
   module ClassMethods
+    def startup!
+      log_managed_entities
+      write_systemd_service_files if MiqEnvironment::Command.supports_systemd?
+      clean_all_workers
+      clean_dequeued_messages
+      purge_report_results
+    end
+
     def log_managed_entities
       region = MiqRegion.my_region
       prefix = "#{_log.prefix} Region: [#{region.region}], name: [#{region.name}]"
       log_under_management(prefix)
       log_not_under_management(prefix)
     end
 
+    def write_systemd_service_files
+      MiqWorkerType.worker_class_names.each do |class_name|
```

Comment:
The worker types are seeded at this point, just to confirm I blew away all of my MiqWorkerType rows and started the server